docs/docs/index.md

```diff
@@ -1,96 +1,148 @@
 - Find pages with no tags for follow-up, or show pretty views of specifically-tagged pages.
-- Create dynamic views which show upcoming birthdays or events, annotated with notes.
+- Create dynamic views which show upcoming birthdays or events recorded in your notes
+
+and many, many things more.
+
+!!! hint "Dataview gives you a fast way to search, display and operate on indexed data in your vault!"
+
+Dataview is highly generic and high performance, scaling up to hundreds of thousands of annotated notes without issue. 
+
+If the built in [query language](query/queries/) is insufficient for your purpose, you can run arbitrary
+JavaScript against the [dataview API](api/intro/) and build whatever utility you might need yourself, right in your notes.
+
+!!! info "Dataview is about displaying, not editing"
+    Dataview is meant for displaying and calculating data. It is no utility to edit data in your source files and will keep your notes always untouched (... except if you're checking a [Task](queries/query-types.md#task-queries) through Dataview.)
+
+## How to Use Dataview
+
+Dataview consists out of two big building blocks: **Data Indexing** and **Data Querying**. 
+
+!!! info "More details on the linked documentation pages"
+    The following sections should give you a broad overview about what you can do with dataview and how. Be sure to visit the linked pages to find out more about the individual parts.
+
+### Data Indexing
+
+Dataview cannot read all your vault, but operates on specific data. Some of your content, like tags and bullet points (including tasks), are [available automatically](annotation/add-metadata.md#implicit-fields) in Dataview. You can add other data through **fields**, either on top of your file [per YAML Frontmatter](annotation/add-metadata.md#frontmatter) or in the middle of your content with [Inline Fields](annotation/add-metadata.md#inline-fields) via the `[key:: value]` syntax. Dataview _indexes_ these data to make it available for you to query. 
+
+!!! hint "Only indexed data is available through Dataview!"
+
+For example, a file might look like this:
+
+```markdown
+---
+author: "Edgar Allan Poe"
+published: 1845
+tags: poems
+---
+
+# The Raven
+
+Once upon a midnight dreary, while I pondered, weak and weary,
+Over many a quaint and curious volume of forgotten lore—
+```
+
+Or like this:
+
+```markdown
+#poems
+
+# The Raven
+
+From [author:: Edgar Allan Poe], written in (published:: 1845)
+
+Once upon a midnight dreary, while I pondered, weak and weary,
+Over many a quaint and curious volume of forgotten lore—
+```
+
+Data-wise, they are identical and only differ in your preffered way how you want to add your [metadata](annotation/add-metadata.md). With this file, you'd have the **metadata field** `author` available and everything Dataview provides you [automatically as implicit fields](annotation/metadata-pages.md), like the tag or note title.
```

Comment:
Typo: preferred

In terms of indexed metadata (or what you can query), they are identical, and only differ in their annotation style. How you want to [annotate your metadata](link) comes down to personal preference.

Second line mostly optional I think but this feels clearer to me.